pkg/k8sutil/k8sutil.go

```diff
@@ -118,7 +118,7 @@ func CreateOrUpdateService(ctx context.Context, sclient clientv1.ServiceInterfac
 		}
 	} else {
 		svc.ResourceVersion = service.ResourceVersion
-		svc.Spec.IPFamily = service.Spec.IPFamily
+		svc.Spec.IPFamilies = service.Spec.IPFamilies
```

Comment:
Have not tested this yet, lets see if tests complain, but I suspect it should be as easy as this 🤞